src/content-helper/dashboard-page/components/posts-table/component.tsx

```diff
@@ -292,42 +287,30 @@ export const PostsTable = ( {
 				{ ! hideHeader && (
 					<thead>
 						<tr>
-							<th className="post-info-header">{ __( 'POST', 'wp-parsely' ) }</th>
+							<th scope="col" className="post-info-header">{ __( 'POST', 'wp-parsely' ) }</th>
 							{ ! compact && (
-								<th className="boost-perf-header">{ __( 'BOOST PERFORMANCE', 'wp-parsely' ) }</th>
+								<th scope="col" className="views-header">
+									{ __( 'VIEWS', 'wp-parsely' ) }
+									<span className="views-header-period">(30 days)</span>
```

Comment:
I think we need to make the `(30 days)` string translatable.

Reply:
Good catch, fixed in https://github.com/Parsely/wp-parsely/pull/3293/commits/b0f374ca7cf1152d8bd67a921a5b515abe14412d!